core/api_tokens.go

```diff
@@ -0,0 +1,47 @@
+package core
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (a *api) createTokens(g *gin.Context) {
+	token, err := a.node.CreateCafeToken()
+	if err != nil {
+		a.abort500(g, err)
+		return
+	}
+	g.String(http.StatusCreated, token)
+}
+
+func (a *api) lsTokens(g *gin.Context) {
```

Comment:
This one (and related cli tools etc) isn't really all that useful tbh. Since the Cafe never stores the actual token (only the bcrypt hashed version) listing them doesn't provide much utility beyond sanity checking the id portion...